R/pkg/R/functions.R

```diff
@@ -2276,9 +2276,8 @@ setMethod("n_distinct", signature(x = "Column"),
             countDistinct(x, ...)
           })
 
-#' @rdname nrow
-#' @param x Column to compute on
-#'
+#' @param x a Column.
+#' @rdname n
```

Comment:
Perhaps count is a better name? I investigated a little more. In the generated docs, each, say \name{count} and \alias{count} can only appear in one doc. Otherwise the check will complain duplicate names or aliases found in 1.rd and 2.rd.